.github/workflows/ci.yml

```diff
@@ -380,8 +381,9 @@ jobs:
         shell: pwsh
         working-directory: webapp
         run: npm run check
-      
-      - name: check recording player
+
+      # Check the recording player here so we don't have to download Biome again in the devolutions-gateway-players job.
+      - name: Check recording player
```

Comment:
suggestion: I think it’s fine to download biome again so the jobs are perfectly separated. Downloading biome is not much, because it’s a binary package behind.

Reply:
updated